entities/weapons/pocket/cl_menu.lua

```diff
@@ -61,7 +61,11 @@ function DarkRP.openPocketMenu()
     reload()
     frame:SetSkin(GAMEMODE.Config.DarkRPSkin)
 end
-usermessage.Hook("PocketMenu", DarkRP.openPocketMenu)
+net.Receive( "DarkRP_PocketMenu", function( len, pl )
+	if not ( IsValid( pl ) and pl:IsPlayer() ) then
```

Comment:
Why does the player need to be valid? It's always LocalPlayer right?

Reply:
I misunderstood the documentation; I thought it would have been possible to receive messages from other players (via some means) and therefore would have wanted to ensure we weren't allowing other people to force the local player to open their pockets. I see that clients can only use SendToServer. Removing it seems to function correctly in both singleplayer and multiplayer so it should be good after this.